logistics_planning_invoicing/models/account_move_line.py

```diff
@@ -0,0 +1,28 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (https://www.gnu.org/licenses/lgpl-3.0.html)
+
+from odoo import models, fields
+
+
+class AccountMoveLine(models.Model):
+    _inherit = 'account.move.line'
+
+    logistics_schedule_id = fields.Many2one('logistics.schedule')
+
+    def create(self, values):
+        res = super().create(values)
+        for record in res.filtered(lambda x: x.logistics_schedule_id):
+            record.logistics_schedule_id.account_move_line_id = record.id
```

Comment:
aghhhh missing return (without it, this shouldn't work):
```suggestion
            record.logistics_schedule_id.account_move_line_id = record.id
        return res
```